website/docs/community/contributing.md

```diff
@@ -12,8 +16,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Contribution Guidelines
-
 **Apache Hadoop Submarine** is an [Apache 2.0 License](https://github.com/apache/submarine/blob/master/LICENSE) Software.
```

Comment:
Need changed `Apache Hadoop Submarine` to `Apache Submarine`.

Reply:
Done